pkg/controller/servicebindingrequest/annotations_test.go

```diff
@@ -105,7 +105,55 @@ func TestAnnotationsSetAndRemoveSBRAnnotations(t *testing.T) {
 	objs = append(objs, u)
 
 	t.Run("SetSBRAnnotations", func(t *testing.T) {
-		err := SetSBRAnnotations(client, namespacedName, objs)
+		originCopy := u.DeepCopy()
```

Comment:
Perhaps `originCopy` can be simplified since `SetSBRAnnotations` is already doing a deep copy.

Reply:
I don't think we can deduce the details of `SetSBRAnnotations` in test cases. So this test case can insure that func `SetSBRAnnotations` is really doing a deep copy.